bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/devices/bridge/BridgeHandler.java

```diff
@@ -432,6 +433,23 @@ public List<Room> getRooms() throws InterruptedException {
         }
     }
 
+    /**
+     * Get public information from Bosch SHC.
+     */
+    public PublicInformation getPublicInformation()
```

Comment:
Don't we already have a method for that somewhere?

Reply:
We not yet have this simple one:
* the BoschHTTPClient can only return the URL with ``getPublicInformationUrl()``
* the BridgeDiscoveryParticipant has the function ``getPublicInformationFromPossibleBridgeAddress`` but uses an own httpClient

Therefor this new function uses the already existing httpClient of the bridgeHandler instance.